tags.go

```diff
@@ -59,6 +66,54 @@ type TagDeleteInput struct {
 	Id ID `json:"id"`
 }
 
+func (client *Client) GetTaggableResource(resourceType TaggableResource, identifier string) (TaggableResourceInterface, error) {
+	var err error
+	var taggableResource TaggableResourceInterface
+
+	if !slices.Contains(AllTaggableResource, string(resourceType)) {
+		return nil, fmt.Errorf("not a taggable resource type: %s" + string(resourceType))
+	}
```

Comment:
couldn't this be acheived by using the `default` case in the below switch statement?

Reply:
Why....... yes. Yes it could 🤯